servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -2835,6 +2835,11 @@ RenderForwardMobile::~RenderForwardMobile() {
 		for (const RID &rid : scene_state.uniform_buffers) {
 			RD::get_singleton()->free(rid);
 		}
+		for (uint32_t i = 0; i < RENDER_LIST_MAX; i++) {
+			if (scene_state.instance_buffer[i] != RID()) {
```

Comment:
```suggestion
			if (scene_state.instance_buffer[i].is_valid()) {
```

Reply:
Seeing as this part is from "render_forward_clustered.cpp", will you be changing that file also?

Reply:
Looks like there's about 90 instances where we could move to `is_valid` or `is_null`. I could take that on in a separate task if it'd be useful.